### DISCUSSION TITLE
Output Entity HA-Name / Config object

### DISCUSSION THREAD

**Discussion by janholtmann** [zachowj/node-red-contrib-home-assistant-websocket#461] [Q&A]
Is there a possibilty to attach the HomeAssistant-Config Name (or alternatively the whole config object) to outgoing messages or reference this another way? I want to use the name in a subsequent notify and haven't found a way yet.
I have started to do a workaround by setting an attribute with the same name but this is very cumbersome as I have 100+ binary sensor entities to do this for.

**Comment by zachowj** [comment]
Currently, there is no way to output the config into the output. If you know the`entity_id` you could use a current-state node to grab the friendly name which would be the name field of the sensor.

This will become available once I switch the entity node to use custom outputs.
